Backend/AMQPBackend.php

```diff
@@ -63,6 +63,11 @@ class AMQPBackend implements BackendInterface
     protected $ttl;
 
     /**
+     * @var null|int
+     */
+    protected $prefetchCount;
```

Comment:
As per our new policy, please make this private, although everything else is protected.